docs/ui/components/CommandMenu/CommandFooter.tsx

```diff
@@ -0,0 +1,62 @@
+import { css } from '@emotion/react';
+import { breakpoints, spacing, theme, typography } from '@expo/styleguide';
+
+import { FOOTNOTE, KBD } from '~/ui/components/Text';
+
+export const CommandFooter = () => (
+  <div css={commandFooterStyle}>
+    <FOOTNOTE css={[footnoteStyle, hideOnMobileStyle]}>
+      <KBD>↵</KBD>
+      <span>to select</span>
+    </FOOTNOTE>
+    <FOOTNOTE css={[footnoteStyle, hideOnMobileStyle]}>
+      <KBD>↑</KBD>
+      <KBD>↓</KBD>
+      <span>to navigate</span>
+    </FOOTNOTE>
+    <FOOTNOTE css={[footnoteStyle, hideOnMobileStyle]}>
+      <KBD>esc</KBD>
+      <span>to close</span>
+    </FOOTNOTE>
+    <FOOTNOTE css={[footnoteStyle, rightSectionStyle]}>
+      <span>Search by</span>
+      <AlgoliaLogo />
+    </FOOTNOTE>
+  </div>
+);
+
+const commandFooterStyle = css({
+  display: 'flex',
+  gap: spacing[4],
+  padding: `0 ${spacing[4]}px`,
+  height: 44,
+  alignItems: 'center',
+});
+
+export const footnoteStyle = css({
+  ...typography.fontSizes[12],
```

Comment:
There's no need to spread the font size since all named text styles like `<FOOTNOTE>` already have responsive font sizes applied.

Reply:
There is no component with 12px size font at least right now.